wls-basisdaten-service/src/main/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/domain/referendumvorlagen/ReferendumvorlagenRepository.java

```diff
@@ -0,0 +1,51 @@
+package de.muenchen.oss.wahllokalsystem.basisdatenservice.domain.referendumvorlagen;
+
+import de.muenchen.oss.wahllokalsystem.wls.common.security.domain.BezirkUndWahlID;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.springframework.cache.annotation.CacheEvict;
+import org.springframework.cache.annotation.CachePut;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.security.access.prepost.PreAuthorize;
+
+@PreAuthorize("hasAuthority('Basisdaten_READ_Referendumvorlagen')")
```

Comment:
```suggestion
@PreAuthorize("hasAuthority('Basisdaten_READ_WLSReferendumvorschlaege')")
```
Falls nicht bewusst abweichend vom Altsystem bitte die Bezeichnung der Repo Authorities überprüfen auch bei WRITE und DELETE

Reply:
besprochen:
- neues Rechte ist lesbarer
- Skripte zur DB-Initialisierung müssen neu geschrieben werden was ein vertretbarer Aufwand ist